pkg/inclusion/paths.go

```diff
@@ -0,0 +1,113 @@
+package inclusion
+
+// genSubTreeRootPath calculates the path to a given subtree root of a node, given the
+// depth and position of the node. note: the root of the tree is depth 0.
+// nolint
```

Comment:
> Use //nolint instead of // nolint because machine-readable comments should have no space by Go convention.

From: https://golangci-lint.run/usage/false-positives/

Proposal to disable the explicit linters that are in violation rather than all linters so that we get linter warnings if a separate violation (that we don't expect) is flagged in this code block.

[Optional] to include a note on why so that maintainers know when this `//nolint` can be removed

```suggestion
// The following nolint can be removed after this function is used.
//nolint:unused,deadcode
```